recipes-azure-iot/azure-identityd/aziot-identityd.inc

```diff
@@ -64,9 +66,7 @@ do_compile() {
 
     sed -i 's/^RELEASE = 0$/RELEASE = 1/'       ${S}/Makefile
     sed -i -e 's/CARGO_TARGET = \(.*\)/CARGO_TARGET = ${TARGET_SYS}/g' ${S}/Makefile
-
-    # for rust toolchain >= 1.82: extend the linker library search path via RUSTFLAGS
-    sed -i -e '/^CARGO =/s/\\$/RUSTFLAGS="$(RUSTFLAGS) -L $(CARGO_OUTPUT_ABSPATH)" \\/' ${S}/Makefile
+    sed -i -e '/^CARGO_OUTPUT_ABSPATH =/ a\export TARGET_BUILD_DIR = $(CARGO_OUTPUT_ABSPATH)' ${S}/Makefile
```

Comment:
is there a difference between `TARGET_BUILD_DIR` and the implicit `CARGO_TARGET_DIR`?
Or is `CARGO_TARGET_DIR` only to be used, if you want to customize the target dir?

Reply:
in case using the cargo environment variable "CARGO_TARGET_DIR", the target build folder will be "<host-path>/target/aarch64-omnect-linux/release/aarch64-omnect-linux/release". The library search path will be search in 
"<host-path>/target/aarch64-omnect-linux/release" and therefor i decided to create an own variable "TARGET_BUILD_DIR"

Reply:
I created a operational task to research that. the nested content of `CARGO_TARGET_DIR` makes no sense to me.